backend/app/controllers/spree/admin/resource_controller.rb

```diff
@@ -82,7 +82,7 @@ def update_positions
       records = model_class.where(id: positions.keys).to_a
 
       positions.each do |id, index|
-        records.find { |r| r.id == id.to_i }&.set_list_position(index)
+        records.find { |r| r.id.to_s == id }&.set_list_position(index)
```

Comment:
Non-blocking, but given we're not testing it, adding a code comment here would be good.

Reply:
I write a com, good idea indeed when two implementations make the spec green and we want one implementation more than the other. 
But my English is not accurate so feel free to suggest something more accurate if needed ;)
Thanks ;)